### PR TITLE
Fix some reportback bugz.

### PR DESCRIPTION
**Pull request by DFurnes**
#### What's this PR do?
This pull request makes some changes to the `/reportbacks` endpoint in Rogue: mainly reformatting the `kudos` object so it's an array with one item in it (weird!), and switching the item's `created_at` to be a UNIX timestamp (also weird!). I'll make a Phoenix PR to address the remaining inconsistencies.

I also removed the unused `ReportbackItemTransformer` because it's confusing to have around!

#### How should this be reviewed?
👀 and :octocat: commits!

#### Any background context you want to provide?
🤷‍♂️

#### Relevant tickets
References DoSomething/phoenix#7456.

#### Checklist
- [ ] Documentation added for new features/changed endpoints.
- [ ] Tested on staging.
- [ ] Pinged a PM if this is a larger PR that would benefit from some additional testing love.

### CODE REVIEW

File: app/Http/Transformers/ReportbackTransformer.php [hunk 1]
```diff
@@ -21,8 +21,7 @@ public function transform(Post $post)
             'id' => $post->id,
             'status' => $post->status,
             'caption' => $post->caption,
-            // Add link to review reportback item in Rogue here once that page exists
-            // 'uri' => 'link_goes_here'
+            'uri' => url(config('services.phoenix.uri') . '/api/v1/reportbacks/'.$post->id, ['absolute' => true]),
```
Comment (katiecrane) [inline app/Http/Transformers/ReportbackTransformer.php]: will it actually live here?

Comment (DFurnes) [inline app/Http/Transformers/ReportbackTransformer.php]: Oh wait, I think it should be `/api/v1/reportback-items` but otherwise yes!

Comment (sbsmith86) [inline app/Http/Transformers/ReportbackTransformer.php]: hmm yea, it I thought this was meant to be like the RBs page in phoenix, which we don't have a parallel for yet in rogue.

Comment (katiecrane) [inline app/Http/Transformers/ReportbackTransformer.php]: yeah with Rogue on I don't think anything will live there (I thiiiink Phoenix would try to grab the reportback that has an rbid that matches `$post->id`??)

Comment (DFurnes) [inline app/Http/Transformers/ReportbackTransformer.php]: I don't think so, unless I'm misreading the [current production response](https://www.dosomething.org/api/v1/reportback-items?campaigns=7656&page=3).

Comment (katiecrane) [inline app/Http/Transformers/ReportbackTransformer.php]: ahhhhhhh I see what it is now 👌

**Pull request merged**